### ISSUE TITLE
Failure to install

### ISSUE THREAD

**Issue by topkock** [kasteleman/com.plugwise.zigbee#3]
Hi,

This app is failing to install.
Message I get is of unknown error ' [invalid checksum]
Trying to install on Homey v2

Thanks

**Comment by kasteleman** [comment]
I have to check.

Currently not on v2 myself.

Regards,

Martin

Op do 24 jan. 2019 om 16:16 schreef topkock <notifications@github.com>

**Comment by kasteleman** [comment]
Just checked, app validates without problem.

Do you have other zigbee apps in use?

Regards,

Martin

**Comment by topkock** [comment]
No I don't.
Just saw little more detail of the error message trying to install via the website. It says there [invalid checksum]: [object Object], maybe that helps?
Thanks

**Comment by kasteleman** [comment]
Ok, committed some changes to my GitHub repo.

Can not test, because not on V2.0

Do you know how to side load a app?

**Comment by topkock** [comment]
still not working, same error.
with side load you mean loading it via command prompt?
not yet but wanted to experiment with that anyway, so will try this weekend.

**Comment by kasteleman** [comment]
Yes by cli. Did not update it in the app store, because i don’t know yet if
the change fixes the problem. Meanwhile found a developer who wants to test
later on this weekend.

I’ll keep you informed! If you manage to try, let me know the output.

Op vr 25 jan. 2019 om 15:10 schreef topkock <notifications@github.com>

**Comment by topkock** [comment]
Managed to install through CLI.
Thanks for the help!

Op 25 jan. 2019 om 15:15 heeft kasteleman <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:

Yes by cli. Did not update it in the app store, because i don’t know yet if
the change fixes the problem. Meanwhile found a developer who wants to test
later on this weekend.

I’ll keep you informed! If you manage to try, let me know the output.

Op vr 25 jan. 2019 om 15:10 schreef topkock <notifications@github.com<mailto:notifications@github.com>>

**Comment by kasteleman** [comment]
Thx for the feedback. Then i can push it to the app store!

Op vr 25 jan. 2019 om 23:46 schreef topkock <notifications@github.com>
